src/NServiceBus.AcceptanceTests/MessageId/When_message_has_no_id_header.cs

```diff
@@ -5,7 +5,7 @@
     using System.Threading.Tasks;
     using AcceptanceTesting;
     using EndpointTemplates;
-    using Pipeline;
+    using NServiceBus.Pipeline;
```

Comment:
yep. Can you verify it works correct on your machine? Do I need to reset too? :D